main.rb

```diff
@@ -0,0 +1,87 @@
+require_relative 'app'
+
+def create_person(app)
+  puts "\n"
+  puts 'Do you want to create a student (1) or teacher (2)? [Input number]:'
+  type = gets.chomp.to_i
+  puts 'Enter person age:'
+  age = gets.chomp.to_i
+  puts 'Enter person name:'
+  name = gets.chomp
+  if type == 1
+    puts 'Has parent permission? [Yes/No]'
+    has_parent_permission = gets.chomp
+    app.create_student(age, name, has_parent_permission)
+  elsif type == 2
+    puts 'Enter specialization'
+    specialization = gets.chomp
+    app.create_teacher(age, specialization, name)
+  else
+    puts 'Invalid input'
+  end
+end
+
+def create_book(app)
+  puts "\n"
+  puts 'Enter book title:'
+  title = gets.chomp
+  puts 'Enter book author:'
+  author = gets.chomp
+  app.create_book(title, author)
+end
+
+def create_rental(app)
+  puts "\n"
+  puts 'Enter rental date:'
+  date = gets.chomp
+  app.list_books
+  puts 'Enter book number:'
+  book = gets.chomp.to_i
+  app.list_people
+  puts 'Enter person number:'
+  person = gets.chomp.to_i
+  app.create_rental(date, app.books[book - 1], app.people[person - 1])
+end
+
+def list_rentals(app)
+  puts "\n"
+  app.list_people
+  puts 'Enter person id:'
+  id = gets.chomp.to_i
+  app.list_rentals(id)
+end
```

Comment:
- [ ] The app logic for implementing the selected options should be moved to the `app.rb` file. This `main.rb` file is solely for presenting the initial options for the users. So kindly move this logic to the `app.rb` file.